public/app/plugins/datasource/influxdb/components/VisualInfluxQLEditor/Editor.tsx

```diff
@@ -112,7 +103,7 @@ export const Editor = (props: Props): JSX.Element => {
 
   return (
     <div>
-      <SectionWrap initialName="from">
+      <Section label="FROM" fill={true}>
```

Comment:
Out of curiosity - is there a reason why we are changing lowercased labels to uppercased? This doesn't seem related to Graphite migration, therefore I would expect just refactor, not changed casing.

Reply:
Influx has these labels in uppercase and the uppercase was applied by default in `SectionWrap` (this component part of Influx code). I moved `SectionWrap` to grafana-ui and called in `Section` to make it more generic, but making labels uppercase by default didn't look right to me. I changed the default behavior of the component to use the label as passed in label property. However, Influx editor displays labels in uppercase (it matches SQL syntax better) so I had to change it.

Reply:
Thanks for context! 🧡